app/src/main/java/com/kms/cura/view/LoginActivity.java

```diff
@@ -25,6 +27,13 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
         initView();
+        loginButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
Should make LoginActivity implements View.OnClickListener() to avoid anonymous class here. Also the setOnClickListener should not be call in the constructor, so maybe move it to where loginButton is initialized.